draft-fossati-tls-attestation.md

```diff
@@ -303,16 +337,74 @@ Auth | {CertificateVerify}
 ~~~~
 {: #figure-background-check-model2 title="TLS Client Providing Evidence to TLS Server."}
 
-# Evidence Extensions (Background Check Model)
+## TLS Client authenticating using results 
+
+In this use case the TLS client, as the attester, provides attestation results
+to the TLS server. The TLS client is the attester and the the TLS server acts as
+a relying party. Prior to delivering its Certificate message, the client must
+contact the verifier (not shown in the diagram) to receive the attestation
+results that it will use as credentials.
```

Comment:
Here again the 2nd sentence duplicates the first one.

Reply:
I'll likely rebase this PR on top of the early exporters one, so will take care of this afterwards.